packages/create-redwood-app/template/redwood.toml

```
@@ -9,6 +9,7 @@
   title = "Redwood App"
   port = 8910
   apiProxyPath = "/.redwood/functions"
+  includeEnvironmentVariables = [] # The array should look like ['SECRET_API_KEY', 'ANOTHER_ONE']
```

Would it be useful to add a comment to link to the documentation

> includeEnvironmentVariables is the set of environment variables to whitelist for the web side

https://redwoodjs.com/docs/app-configuration-redwood-toml#includeenvironmentvariables

---

I don't see why not apart from the extra comments, but they can be removed so 🤷🏾‍♂️

---

```suggestion
  includeEnvironmentVariables = [] # The array should look like ['ENV_VAR_AVAILBLE_TO_WEB_SIDE', 'ANOTHER_ONE']
```
I think we should make it clear that we're not trying to share a secret here, but making it available to the web side (where we don't make it available by default. precisely because they're secrets)